com.vscodium.codium.desktop

```diff
@@ -0,0 +1,18 @@
+[Desktop Entry]
+Name=VSCodium
+Comment=Code Editing. Redefined.
```

Comment:
The desktop file & metainfo should be ideally upstreamed.

Reply:
@bilelmoussaoui, thanks for the review. I'll suggest this to the @stripedpajamas